### PR TITLE
added integration to load to mini games

### PR DESCRIPTION
**Pull request by bazahmed**

### CODE REVIEW

File: Master Project/Assets/Scenes/Main/IngredientDirectiveScript.cs [hunk 1]
```diff
@@ -27,17 +28,16 @@ public class IngredientDirectiveScript : MonoBehaviour {
 		//Debug.Log ("Collision");
 		if (col.gameObject.tag == "ItemOne") {
```
Comment (noahwillcrow) [inline Master Project/Assets/Scenes/Main/IngredientDirectiveScript.cs]: None of this is in a method... This won't compile.

File: Master Project/Assets/Scenes/Main/IngredientDirectiveScript.cs [hunk 2]
```diff
@@ -27,17 +28,16 @@ public class IngredientDirectiveScript : MonoBehaviour {
 		//Debug.Log ("Collision");
 		if (col.gameObject.tag == "ItemOne") {
 			Debug.Log ("first");
+			SceneManager.LoadScene ("Shaking");
 		}
 		else if (col.gameObject.tag == "ItemTwo") {
 			Debug.Log ("second");
+			SceneManager.LoadScene ("Microwave_Goldberg");
```
Comment (noahwillcrow) [inline Master Project/Assets/Scenes/Main/IngredientDirectiveScript.cs]: This hard coding is definitely not a long term solution. There is an enum IngredientType (currently only has WhippedCream). What should be done is there should be a Dictionary<IngredientType ingredientType, string SceneName> that maps each ingredient to the proper scene.

Also, each scene is going to need information about what ingredient they're working on.

File: Master Project/ProjectSettings/ProjectVersion.txt [hunk 3]
```diff
@@ -1 +1 @@
-m_EditorVersion: 2017.4.12f1
+m_EditorVersion: 2017.4.10f1
```
Comment (noahwillcrow) [inline Master Project/ProjectSettings/ProjectVersion.txt]: Wrong version of Unity. We've moved to 2017.4.12f1

File: Master Project/Assets/Scenes/Main/IngredientDirectiveScript.cs (hunk 1 above)
Comment (bazahmed) [reply Master Project/Assets/Scenes/Main/IngredientDirectiveScript.cs]: Woah that’s pretty weird it’s in a method for me idk why it’s doing that

**Pull request merged**